docker02.txt

```diff
@@ -0,0 +1,3 @@
+docker load -i .\hello-world.tar
+docker run --name hello-world -p 5000:5000 hello-world:latest
+docker start hello-world
```

Comment:
no need to use docker run and docker start, "start" is for existing containers, "run" also creates them.